.eslintrc.json

```diff
@@ -4,7 +4,13 @@
         "es2021": true,
         "jest": true
     },
-    "extends": ["plugin:react/recommended", "airbnb"],
+    "extends": [
+        "plugin:react/recommended",
+        "airbnb",
```

Comment:
plugins can override each other's rules, so we need to make sure that i.e `airbnb ` is the main one and all rules that are overriden are taken from it. I'm not sure where exactly should `airbnb` be in the `extends` list though (end to beginning), this needs to be investigated